types/js2coffee/index.d.ts

```diff
@@ -0,0 +1,396 @@
+// Type definitions for JS2Coffee 2.2
+// Project: https://github.com/js2coffee/js2coffee
+// Definitions by: Munin M. <https://github.com/SNDST00M>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 4.1
```

Comment:
Do you want this specific version? Native JS packages are not most modern JS implementations

Reply:
https://coffeescript.org/#compatibility I assumed from this that the JS used is a modern lib

Reply:
ok, let me rephrase. By putting at 4.1, you're potentially limit this package interoperability with other definitions, as it requires 4.1 version. What's the purpose of this version here?